### DISCUSSION TITLE
Roadmap infos

### DISCUSSION THREAD

**Discussion by toonetcreation** [akeeba/panopticon#831] [General]
Hi,

First of all I would like to send a huge thanks to Nicholas about the huge work you did on panopticon.
I'm very interested to test it, the philosophy behind the scene looks much more better rather than the tool I'm using.
I have just 2 questions :

1) is it possible to uninstall an extension?

2) what is the future of panopticon ? I mean : as the product is free, the product will be maintened next year and after ? what is on your mind about that?

Thanks again
L.

**Comment by nikosdion** [comment]
Thank you for your kind words!

I have deliberately NOT included uninstalling an extension, and I am not going to consider it as a feature. First of all, it is very dangerous when you make it so easy to take a destructive action on a site. People _will_ break their sites and blame the tool on the screen, not the tool sitting in front of the screen. Moreover, some extension developers _assume_ that their extension is installed or uninstalled from within the Joomla! backend with a Super User logged in (none of which is a given since Joomla! 3.2 released more than twelve years ago, by the way). This would break the uninstallation. It's the same reason why the promised feature of mass installing extensions across sites has not been implemented yet.

Regarding the future, I cannot make a promise because I don't know what the future holds. Assuming I still have my good health and can make a living out of my work with Joomla! and WordPress extensions, my _intention_ is to keep maintaining Panopticon. As far as I am concerned, Panopticon is an extension to Akeeba Backup Professional and Admin Tools Professional much like Kickstart, Remote CLI, and UNiTE are.

**Reply by toonetcreation** [reply]
Thanks for your feedback.
About uninstall, I agree, make sense.
At the end, this is possible to uninstall an extension thanks to joomla CLI so, there is a solution.
About mass install file, the same, can be done with joomla CLI, so that's fine.
Ok for the future, I stay tuned and I will test the tool ;-)
